README.md

```diff
@@ -1237,6 +1253,27 @@ Other Style Guides
     }
     ```
 
+  <a name="classes--members-order"></a>
+  - [9.7](#classes--members-order) Always use this order for class members
+    1. static properties
+    1. constructor
```

Comment:
@dekdekbaloo well Github markdown works for having 1 in all list items, for more official docs, lets have proper list numbers 1, 2, 3

Reply:
I saw from the original version using 1. all the way down in case there are additions in later time so we don't have to recall the number and mess with that when re-ordering stuffs, that's why I did it this way.

Anyway, as this is pretty small, I'll make change based on your suggestion.